src/styles/index.scss

```diff
@@ -1,3 +1,142 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.card {
+  background: #fff;
+  width: 198px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin-top: 32px;
+    margin-left: 19px;
+  }
+
+  &__title {
+    width: 166px;
+    height: 36px;
+    margin-top: 40px;
+    margin-left: 16px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product {
+    height: 14px;
+    margin-top: -7px;
```

Comment:
It is not a good idea to use negative margins. It works and you can use in some cases, but do not over-use it, specially if it is possible to get the styling you need without it